CHANGELOG.md

```diff
@@ -34,6 +35,100 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
+
+We expect that this release candidate is quite close to the final v1.1.0
+release. However, breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.0.0:
+
+## Standard Channel
+
+### GRPCRoute has Graduated to GA 🎉
+
+GRPCRoute has graduated to GA (v1) and is now part of the Standard Channel. If
+you are already using the experimental version GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: @gnossen
+
+### Mesh Support
+
+The standard for using Gateway API for Mesh has formally graduated to v1.0.
+
+Leading Contributors: @howardjohn, @keithmattix, @kflynn
```

Comment:
@howardjohn @keithmattix @kflynn can you help fill this out?

Reply:
Yep - I want to make sure we add @mikemorris who's also been pushing this forward since the early days

Reply:
Great idea @keithmattix! Mind making a suggestion to formally make the change? @howardjohn just added some initial text here, feel free to update or revise that too.